backend/cache/cache.go

```diff
@@ -381,16 +385,16 @@ func NewFs(name, rootPath string) (fs.Fs, error) {
 		}
 	}()
 
-	if doDirChangeNotify := wrappedFs.Features().DirChangeNotify; doDirChangeNotify != nil {
-		doDirChangeNotify(f.receiveDirChangeNotify, f.chunkCleanInterval)
+	if doChangeNotify := wrappedFs.Features().ChangeNotify; doChangeNotify != nil {
+		doChangeNotify(f.receiveChangeNotify, f.chunkCleanInterval)
 	}
 
 	f.features = (&fs.Features{
 		CanHaveEmptyDirectories: true,
 		DuplicateFiles:          false, // storage doesn't permit this
 	}).Fill(f).Mask(wrappedFs).WrapsFs(f, wrappedFs)
 	// override only those features that use a temp fs and it doesn't support them
-	f.features.DirChangeNotify = f.DirChangeNotify
+	//f.features.ChangeNotify = f.ChangeNotify
```

Comment:
Is this intended?  It looks like cache no longer supports ChangeNotify?

Reply:
It just supports it when the wrapped fs does too. Cache doesn't provide notifications on its own and tests (at the moment only those) fail when it wraps a remote that doesn't support it.

Perhaps it will in the future, regardless of the wrapped remote, but at the moment to just add notifications on every change can likely trigger a notification mess so better to play it safe.

Reply:
Oh, I see!